docs/setup-cluster/deploy-cluster/on-prem/linux-packages.rst

```diff
@@ -135,74 +152,81 @@ Master and Agent
  Configure and Start the Cluster
 *********************************
 
-#. Ensure that an instance of PostgreSQL is running and accessible from the machine where the master
-   will be run.
+.. important::
```

Comment:
I feel like changing this and the one below to admonitions rather breaks the flow of the steps. I also don't really see the need in the first place, especially for this first one, which is a perfectly good step in itself. The second one about the config values fits a bit better as an admonition, but then it and the subsequent code block should be indented so that they actually appear as part of the step, like the note shortly afterward.

Reply:
resolved